asserts/assertstest/assertstest.go

```diff
@@ -266,7 +273,16 @@ type StoreStack struct {
 }
 
 // NewStoreStack creates a new store assertion stack. It panics on error.
-func NewStoreStack(authorityID string, rootPrivKey, storePrivKey asserts.PrivateKey) *StoreStack {
+// optional privKeys can be in order: root, store, generic
+func NewStoreStack(authorityID string, privKeys ...asserts.PrivateKey) *StoreStack {
```

Comment:
done, and yes I rechecked, I follow up could remove passing keys at all in a few places